src/LibraryManager/Providers/jsDelivr/JsDelivrCatalog.cs

```diff
@@ -29,16 +31,24 @@ internal class JsDelivrCatalog : ILibraryCatalog
         private readonly string _providerId;
         private readonly ILibraryNamingScheme _libraryNamingScheme;
         private readonly ILogger _logger;
-        private readonly IWebRequestHandler _webRequestHandler;
-
-        public JsDelivrCatalog(string providerId, ILibraryNamingScheme namingScheme, ILogger logger, IWebRequestHandler webRequestHandler, INpmPackageInfoFactory packageInfoFactory, INpmPackageSearch packageSearch)
+        private readonly ICacheService _cacheService;
+        private readonly string _cacheFolder;
+
+        public JsDelivrCatalog(string providerId,
```

Comment:
I wonder if you have considered, for consistency with the `CdnsCatalog` class, to pass the `jsDelivrProvider` in place of `providerId` and `cacheFolder`?  Same can be said for the `UnpkgCatalog` class.

Reply:
Actually, at some point I'm planning to do the opposite: make CdnjsCatalog take simple values instead of the provider.  This better decouples the two types, and would simplify construction of the CdnjsCatalog during testing as we would no longer have to supply a CdnjsProvider with all of _its_ dependencies.  But that's out of the scope of this change...